glean-core/src/lib.rs

```diff
@@ -89,18 +122,32 @@ impl Glean {
         self.upload_enabled
     }
 
+    /// Get the application ID as specified on instantiation.
     pub fn get_application_id(&self) -> &str {
         &self.application_id
     }
 
+    /// Get the data path of this instance.
     pub fn get_data_path(&self) -> &Path {
         &self.data_path
     }
 
+    /// Get a handle to the database.
     pub fn storage(&self) -> &Database {
         &self.data_store
     }
 
+    /// Take a snapshot for the givven store and optionally clear it.
+    ///
+    /// ## Arguments
+    ///
+    /// * `store_name` - The store to snapshot.
+    /// * `clear_store` - Whether to clear the store after snapshotting.
+    ///
+    /// ## Return value
+    ///
+    /// Returns the snapshot as a JSON string.
```

Comment:
```suggestion
    /// Returns the snapshot in a string encoded as JSON.
```

"JSON string" is a bit ambiguous -- it could mean [one of these](https://docs.serde.rs/serde_json/enum.Value.html#variant.String)